app/views/admin/edition_images/preview.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for :context, "#{@edition.title}" %>
+<% content_for :page_title, "Preview image" %>
+<% content_for :title, "Preview image" %>
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: admin_edition_images_path(@edition)
+  } %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body"> The image you selected is exactly the right size. The whole image will be used. </p>
+    <img src="<%= @image.url %>" height="640" alt="Image preview" class="app-view-edition-images__image govuk-!-margin-bottom-4"><br>
```

Comment:
For the sake of consistency, I think we should either include both `width` and `height` attributes, or neither.

It seems safe enough to hardcode these values, given they're also hardcoded in the 'Image information' sidebar on this page.